.github/workflows/coverage.yml

```diff
@@ -36,9 +36,8 @@ env:
 
 jobs:
   coverage:
-    name: Coverage nightly
+    name: Coverage on stable
```

Comment:
> > Leaving this comment here so we don't forget about the patch file.
> 
> I looked at `coverage.patch.yml` and it's a no-op, no Rust toolchain to update

The job name in the patch file needs to be changed, so we can require the job in branch protection rules, but skip it when it is not needed:
https://github.com/ZcashFoundation/zebra/blob/e4ca6c358ec9807c5c97e610d51297a0dab7996d/.github/workflows/coverage.patch.yml#L16